react-vending-machine/src/components/Menu/index.tsx

```diff
@@ -21,10 +20,37 @@ export default function Menu({
   return (
     <StyledMenu lightOn={lightOn}>
       {icon}
-      <div>{price.toLocaleString("en")}원</div>
+      <div>{price.toLocaleString("ko")}원</div>
```

Comment:
👮‍♂️

![image](https://user-images.githubusercontent.com/57928612/175805236-a7794b63-5dc3-42f4-9b9c-700c7ee9ea83.png)

Reply:
(도리와 월터 리뷰어님과의 즐거운 👮‍♂️ 추억...)

```suggestion
      <div>{price.toLocaleString("ko-KR")}원</div>
```

부분을 말하려고 그랬습니다...ㅎ